spec/namespaces/ml.yaml

```diff
@@ -102,6 +102,31 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/ml.search_models@200'
+  /_plugins/_ml/_predict/{algorithm_name}/{model_id}:
+    post:
+      operationId: ml.predict.0
+      x-operation-group: ml.predict
+      description: Predicts new data with trained model.
+      parameters:
+        - $ref: '#/components/parameters/ml.predict::path.algorithm_name'
+        - $ref: '#/components/parameters/ml.predict::path.model_id'
+      requestBody:
+        $ref: '#/components/requestBodies/ml.predict'
+      responses:
+        '200':
+          $ref: '#/components/responses/ml.predict@200'
+  /_plugins/_ml/_train/{algorithm_name}:
+    post:
+      operationId: ml.train.0
+      x-operation-group: ml.train
+      description: Trains a model synchronously.
+      parameters:
+        - $ref: '#/components/parameters/ml.train::path.algorithm_name'
+      requestBody:
+        $ref: '#/components/requestBodies/ml.train'
+      responses:
+        '200':
+          $ref: '#/components/responses/ml.train@200'
```

Comment:
Add please `POST /_plugins/_ml/_train_predict/{algorithm_name}`

Reply:
Yes, I'll add that API as well.